src/components/Switch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class Switch extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chosenParamIndex: 0,
+        };
+    }
+    handleClick(which) {
+        this.setState({chosenParamIndex: which })
+    }
+
+    getClass(index) {
+        return "Switch__Param " + (this.state.chosenParamIndex === index ? "Switch_activeParam" : "");
+    }
+
+    render() {
+        const { params, text } = this.props;
+        return (
+            <div className="Switch">
+                {text}
+                <span className={this.getClass(0)} onClick={() => this.handleClick(0)}>
+                    {params[0]}
```

Comment:
What params[0] and params[1] means? Maybe it is better to use array destructing and give some meaningful names to array items.

Reply:
The point is to make a universal component which has no hardcoded parameters. In this case there may be paramA, paramB, param0, param1 or params[0], params[1]. If you think that those may be called better, what would you suggest in this case?